### ISSUE TITLE
bash: cannot set terminal process group (-1): Inappropriate ioctl for device

### ISSUE THREAD

**Issue by iamxiaojianzheng** [oysandvik94/curl.nvim#36]
My environment is `git bash` + `neovim 0.10.1`, is it possible to solve this problem?

```
bash: cannot set terminal process group (-1): Inappropriate ioctl for device bash: no job control in this shell bash: "curl https://jsonplaceholder.typicode.com/todos/1 -p -sSL": No such file or directory 
```

**Comment by oysandvik94** [comment]
I'll take a look!

**Comment by oysandvik94** [comment]
@iamxiaojianzheng Can you try it with quotes around the URL? As in:
`curl "https://jsonplaceholder.typicode.com/todos/1" -p`

**Comment by iamxiaojianzheng** [comment]
> @iamxiaojianzheng Can you try it with quotes around the URL? As in:您可以尝试在 URL 周围加上引号吗？如： `curl "https://jsonplaceholder.typicode.com/todos/1" -p`

![image](https://github.com/user-attachments/assets/c6b67059-786e-4209-ba09-ea5c3c193862)

**Comment by iamxiaojianzheng** [comment]
I can use `rest.nvim` and `kulala.nvim` normally. 

I guess this is all a wrapper around the curl command. Is there anything different about `curl.nvim` ?

**Comment by oysandvik94** [comment]
That's right, it's using your underlying shell which is why it proably doesnt work. Those other plugins probably use libraries which are supported regardless of shell.

I'm guessing you get the same error if you just paste the curl command in to git bash?

**Comment by iamxiaojianzheng** [comment]
no problem in gitbash.

![image](https://github.com/user-attachments/assets/d2746964-78fb-4431-a3c3-168ddeef6370)

**Comment by oysandvik94** [comment]
Could you try with the "-sSL" option as well?

**Comment by iamxiaojianzheng** [comment]
Result like this.

![image](https://github.com/user-attachments/assets/bed4aa3d-1588-44ef-89db-af3d19322b1d)

**Comment by oysandvik94** [comment]
Interesting, thanks! I'll try and set it up on my windows PC to debug.

**Comment by iamxiaojianzheng** [comment]
> Interesting, thanks! I'll try and set it up on my windows PC to debug.

This is a strange question. 😂

**Comment by oysandvik94** [comment]
Fixed in #39
